core/Pos/Binary/Class.hs

```diff
@@ -67,6 +67,7 @@ import           Universum
 
 import           Formatting                  (formatToString, int, (%))
 import           Data.Bits                   (Bits (..))
+import qualified Data.Set                as S
```

Comment:
Ah, because stylish is broken on this file, right. I forgot. Well, align it manually then :)